{"componentChunkName":"component---src-pages-content-articles-chained-exceptions-in-java-index-md","path":"/content/articles/chained-exceptions-in-java/","result":{"pageContext":{"frontmatter":{"templateKey":"article","published":true,"displayComments":true,"author":"narramadan","title":"Chained Exceptions in Java","description":"In this article, weâ€™ll have a very brief look at what Exception is and go in depth about discussing the chained exceptions in Java","date":"2019-12-17T23:46:37.121Z","updated":"2019-12-17T23:46:37.121Z","cover":"./preview.jpg","category":"java","tags":["java","exceptions","featured"]}}}}